tox.ini

```diff
@@ -92,6 +93,9 @@ setenv =
     dev: ALEMBIC_CONFIG = {env:ALEMBIC_CONFIG:conf/alembic.ini}
     dev: JWE_SECRET_LMS = {env:JWE_SECRET_LMS:SUPER_SECRET}
     tests: COVERAGE_FILE = {env:COVERAGE_FILE:.coverage.{envname}}
+    dev: DATABASE_URL = {env:DATABASE_URL:postgresql://postgres@localhost/postgres}
+    tests: DATABASE_URL = {env:DATABASE_URL:postgresql://postgres@localhost/h_tests}
+    functests: DATABASE_URL = {env:DATABASE_URL:postgresql://postgres@localhost/h_functests}
```

Comment:
The `sqlalchemy.url` / `DATABASE_URL` setting moved from the config files into an envvar.